### PR TITLE
Add Resource Instance Name in final summary step, #628

### LINKED ISSUE #628: Workflow summary: Show name of instances created in addition to model type
The summary page provides a link to the instance(s) created in the workflow, but only shows the model type.  We should also include the instance name.  In the example below, the "Created or Modified Resource Instance" list should show instance using the following pattern "Instance Name (model type)"

### PR DESCRIPTION
**Pull request by njkim**
Add Resource Instance Name in final summary step, #628

### CODE REVIEW

File: arches_her/media/js/views/components/workflows/upload-document-step.js [hunk 1]
```diff
@@ -115,6 +109,23 @@ define([
             params.resourceid(ko.unwrap(this.consultationResourceId));
             params.workflow.resourceId(ko.unwrap(this.consultationResourceId));
     
+            params.defineStateProperties = function(){
```
Comment (chiatt) [inline arches_her/media/js/views/components/workflows/upload-document-step.js]: defineStateProperties should probably just be removed here

**Review by chiatt — CHANGES_REQUESTED** [review]

File: arches_her/media/js/views/components/workflows/upload-document-step.js (hunk 1 above)
Comment (njkim) [reply arches_her/media/js/views/components/workflows/upload-document-step.js]: The branch is updated the most recent master as of 12/10 and some minor fixe was added.

Comment (njkim) [reply arches_her/media/js/views/components/workflows/upload-document-step.js]: This is ready to be reviewed again.

**Review by chiatt — APPROVED** [review]
Looks good to me.

**Pull request merged**